server/package.json

```diff
@@ -27,9 +27,7 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./apps/immich/test/jest-e2e.json",
     "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js",
-    "api:typescript": "rm -rf ../web/src/api/open-api && npx openapi-generator-cli generate -g typescript-axios -i ./immich-openapi-specs.json -o ../web/src/api/open-api",
-    "api:dart": "rm -rf ../mobile/openapi && npx openapi-generator-cli generate -g dart -i ./immich-openapi-specs.json -o ../mobile/openapi",
-    "api:generate": "npm run api:typescript && npm run api:dart"
+    "api:generate": "./bin/generate-open-api.sh"
```

Comment:
I'm unsure about this change. On the plus side, always generating mobile+web leaves less room for errors. But it also takes longer and is unnecessary if you are only developing mobile or web at a time.
@alextran1502 Do you like to use the individual API generate commands?

Reply:
I currently run `make api`, which does both even though I only develope for web.

I'm happy to make it possible to call them separately if this is inconvenient or takes too long. For me personally it is pretty quick, not to mention I don't actually use it very often. 

On the plus side they would always be in sync this way.

Main motivation was to have all the generator code in one place, but I could add some arg processing to do web, mobile, or both.